lib/rules/lint-no-whitespace-for-layout.js

```diff
@@ -0,0 +1,22 @@
+// lint-no-whitespace-for-layout.js
+'use strict';
+const Rule = require('./base');
+
+module.exports = class noWhitespaceForLayout extends Rule {
+  visitor() {
+    return {
+      TextNode(node) {
+        let source = this.sourceForNode(node);
+        let matches = source.match(/(( )|(\&nbsp\;))(( )|(\&nbsp\;))/g);
+        if (matches !== null) {
+          this.log({
+            message: 'Excess space detected',
```

Comment:
Can you store this error message in a constant `ERROR_MESSAGE` and export it for use in the tests cases to avoid duplication? Many other rules follow that pattern.